docs/implementation.html

```diff
@@ -40,9 +40,9 @@
 
 The goal is to expose all the producer functionality through a single API to the client.
 
-The new producer -
+The Java producer
```

Comment:
I'm wondering if it's really necessary to mention `Java`. It's the only producer that we mention in this documentation, so can't we just call it the "Kafka Producer"?